Lib/inspect.py

```diff
@@ -3296,6 +3297,28 @@ def signature(obj, *, follow_wrapped=True, globals=None, locals=None, eval_str=F
                                    globals=globals, locals=locals, eval_str=eval_str)
 
 
+class BufferFlags(enum.IntFlag):
```

Comment:
I am big fan of `@enum.unique` for these kind of flags, but other stdlib modules do not use it :(

Reply:
Will add it, thanks for the suggestion.

Reply:
We actually can't because there are some aliases. I think those are useful to have because they reflect the C API.